tests/prague/eip7692_eof_v1/eip3540_eof_v1/test_example_valid_invalid.py

```diff
@@ -384,6 +391,874 @@
             EOFException.UNDEFINED_INSTRUCTION,
             id="jump_jumpdest_fails",
         ),
+        pytest.param(
```

Comment:
We discussed with @marioevz and @danceratopz that this array with tests is not great. I don't mind some much the array but maybe it should be a separate test?

But what is bad that we add a ton of sophisticated tests to "example" category.

You should also add test cases to the EOF tracker file.

Reply:
I personally find it confusing when such an array is stored in another file, but here it is quite transparent

Reply:
I think the problem is that `tests/prague/eip7692_eof_v1/eip3540_eof_v1/test_example_valid_invalid.py` contains too many unrelated test cases.

For example it tries to cover many of the deprecated opcodes but not all of them.

We need to break up `test_example_valid_invalid` into different coherent test functions testing similar things, it's too big, and it's not coherent with what it's testing (also no longer an example, it contains multiple valid tests).